src/web/app/pages-static/request-page/instructor-request-form/instructor-request-form.component.ts

```diff
@@ -0,0 +1,93 @@
+import { Component, EventEmitter, Output } from '@angular/core';
+import { FormControl, FormGroup, Validators } from '@angular/forms';
+import { InstructorRequestFormData } from './InstructorRequestFormData';
+
+// eslint-disable-next-line
+const URL_REGEX = /(https?:\/\/)?(www\.)[-a-zA-Z0-9@:%._\+~#=]{2,256}\.[a-z]{2,4}\b([-a-zA-Z0-9@:%_\+.~#?&//=]*)|(https?:\/\/)?(www\.)?(?!ww)[-a-zA-Z0-9@:%._\+~#=]{2,256}\.[a-z]{2,4}\b([-a-zA-Z0-9@:%_\+.~#?&//=]*)/;
+
+@Component({
+  selector: 'tm-instructor-request-form',
+  templateUrl: './instructor-request-form.component.html',
+  styleUrls: ['./instructor-request-form.component.scss'],
+})
+export class InstructorRequestFormComponent {
+
+  arf = new FormGroup({
+    name: new FormControl('', [Validators.required]),
+    institution: new FormControl('', [Validators.required]),
+    country: new FormControl('', [Validators.required]),
+    email: new FormControl('', [Validators.required, Validators.email]),
+    homePage: new FormControl('', [Validators.pattern(URL_REGEX)]),
+    comments: new FormControl(''),
+  }, { updateOn: 'submit' });
+
+  // Create members for easier access of arf controls
+  name = this.arf.controls.name;
+  institution = this.arf.controls.institution;
+  country = this.arf.controls.country;
+  email = this.arf.controls.email;
+  homePage = this.arf.controls.homePage;
+  comments = this.arf.controls.comments;
+
+  hasSubmitAttempt = false;
+
+  @Output() requestSubmitted = new EventEmitter<InstructorRequestFormData>();
```

Comment:
```suggestion
  @Output() requestSubmittedEvent = new EventEmitter<InstructorRequestFormData>();
```

I think we usually have `Event` at the end of the name. You can also call it `submitRequestEvent`, or `requestSubmissionEvent`.